packages/shorebird_cli/lib/src/commands/preview_command.dart

```diff
@@ -143,12 +147,51 @@ class PreviewCommand extends ShorebirdCommand {
       return ExitCode.software.code;
     }
 
+    final completeRelease = allReleases.firstWhereOrNull(
+      (r) => r.version == releaseVersion,
+    );
+    if (completeRelease != null) {
```

Comment:
I don't think that this is necessary, the set of releases with previewable artifacts should be a strict subset of the set of all releases